source/_includes/common-tasks/network_storage.md

```diff
@@ -56,7 +64,20 @@ Protocol:
 {% endconfiguration_basic %}
 
 <sup>1</sup> _Options prefixed with `[NFS]` is only available for NFS targets._<br>
-<sup>2</sup> _Options prefixed with `[CIFS]` is only available for CIFS targets._
+<sup>2</sup> _Options prefixed with `[CIFS]` is only available for CIFS targets._<br>
+<sup>3</sup> _For the `CIFS` option only version 2.1+ is supported._<br>
+<sup>4</sup> _Guest access is not supported, you need to supply a username and password to access the share._<br>
+
+##### Usage types
+
+{% configuration_basic "hassio.network_share.usage" %}
+Backup:
+  description: This will become a target you can use in service calls or when manually creating a backup, the first storage you add with this type will become your new default target. If you want to change the default target, [check out the documentation for that below](#change-default-backup-location).
```

Comment:
```suggestion
  description: This will become a target you can use in service calls or when manually creating a backup. The first storage you add of this type becomes your new default target. If you want to change the default target, [check out the documentation below](#change-default-backup-location).
```